### ISSUE TITLE
Hello! In addition to generating uncertain labels for cars, I also generated uncertain labels for pedestrians and bicycles. After training, the AP value of the car has not changed much, but the AP value of the pedestrian and bicycle has decreased a lot. I want to ask what causes this?

### ISSUE THREAD

**Issue by 6Superman6** [Eaphan/GLENet#5]
Hello! In addition to generating uncertain labels for cars, I also generated uncertain labels for pedestrians and bicycles. After training, the AP value of the car has not changed much, but the AP value of the pedestrian and bicycle has decreased a lot. I want to ask what causes this? @Eaphan

**Comment by Eaphan** [comment]
Can you show me your code by submitting a pull request? So I can help to check the code online.

Besides, I have a suggestion. You can try to tune the coefficients of sub-items of regression loss. Such as:
        loc_loss = loc_loss_src_mean + loc_loss_square_mean + loc_loss_log_mean
====>>>>>
        loc_loss = loc_loss_src_mean + (loc_loss_square_mean + loc_loss_log_mean) * weight
weight can be 0.1 or 0.01.
